pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbcluster/fullalloydbcluster/create.yaml

```diff
@@ -51,3 +51,12 @@ spec:
   encryptionConfig:
       kmsKeyNameRef: 
         name: kmscryptokey-${uniqueId}
+
+  maintenanceUpdatePolicy:
```

Comment:
Similarly, have you verified the test resource has the maintenance window configured properly using this YAML?

Reply:
Verified - https://screenshot.googleplex.com/7efc79A6qNBioFB